genesyscloud/journey_outcome_predictor/genesyscloud_journey_outcome_predictor_init_test.go

```diff
@@ -0,0 +1,47 @@
+package journey_outcome_predictor
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"sync"
+	gcloud "terraform-provider-genesyscloud/genesyscloud"
+	"testing"
+)
+
+var (
+	providerDataSources map[string]*schema.Resource
```

Comment:
You can probably delete this variable since there's no data sources used

Reply:
I tried to remove this variable but it causes the test to fail because it is needed for the function `GetProviderFactories`